actions/postgresql_query_select_action.go

```diff
@@ -50,7 +56,18 @@ func (a *postgresqlQuerySelectAction) Type() string {
 
 // Run runs an Action and returns output and error.
 func (a *postgresqlQuerySelectAction) Run(ctx context.Context) ([]byte, error) {
-	connector, err := pq.NewConnector(a.params.Dsn)
+	tmpDir := filepath.Join(os.TempDir(), fmt.Sprintf("pg_action_%05d", rand.Int63n(99999))) //nolint:gosec
```

Comment:
Can it hit same number which already exists from rand?

Reply:
it can, @percona-csalguero what is the reason to create another temporary directory? we already do it here https://github.com/percona/pmm-agent/blob/2fe2efa823ed2725b77eb4c8d293b8488c1a7299/utils/templates/template.go#L59

Reply:
here and in other places

Reply:
Please check how it was implemented for mongodb